releng/org.eclipse.n4js.releng.utils/src/org/eclipse/n4js/runconfig/CreateConfigFileForSettingDefaultProduct.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2017 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.runconfig;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+
+/**
+ * This class access the variable ${target_home} and create a config.ini file specifying
+ * org.eclipse.n4js.product.product as the default product to run. This saves us from having to manually configure the
+ * product again and again whenever we trigger a test. GH-307. Attention: If you change this class, make sure to make a
```

Comment:
- omit the GH-307
- Add a \<p> before "Attention"
- consider to use <b>Attention:</b>
- ... make sure to _create_ ...